common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -2147,7 +2147,6 @@ schematic = (function() {
 	    this.bg_image.height = this.height;
 
 	    if (!this.diagram_only) {
-		this.canvas.tabIndex = 1; // so we get keystrokes
```

Comment:
The `tabindex` is unnecessary since screenreader users are able to interact with the canvas through their normal browsing commands.